scarlin90-belfast.md

```diff
@@ -0,0 +1,10 @@
+# Belfast, Bel
+
+## Places to Eat
+- Boojum :buritos and :nachos
+- Laverys for :beers
+- Made In Belfast for :dinner
+
+
+## Places to Visit
+
```

Comment:
I've never been, I really like the craft beer scene, are there any breweries that have good tours?